daemon/daemon_test.go

```diff
@@ -703,8 +703,6 @@ func TestListEvents(t *testing.T) {
 		count        int
 		err          string
 	}{
-		//{name: "Normal", count: 1},
```

Comment:
I think it is dangerous just to remove them. The big question is why are they commented out?

Reply:
The goroutines which are used in getServer  and grpc.DialContext creates problem somehow, I cannot say how but when I try to run the test gorutouni nill pointer memory error happened.

Reply:
Instead of removing tests, I believe we should try to understand why they fail. So diagnosing this is important from my perspective.

Reply:
I will try to fix that 👍🏻 by synchronising the goroutines. :)

Reply:
Apparently, getServer function which includes handler, creates problem when creationTime value added to listevents, I will try to find a solution to it.

Reply:
fixed